TwitchChannelPointsMiner/utils.py

```diff
@@ -8,7 +8,7 @@
 from TwitchChannelPointsMiner.constants.browser import USER_AGENTS
 
 
-def millify(input, precision=2):
+def Millify(input, precision=2):
```

Comment:
The entire code base use the first letter in uppercase only for **Class** and not for **methods**.
So please change name or think up something else. For me also `_millify` It's good, but please don't use capital letter

Reply:
Or `millify_numbers`

Reply:
> The entire code base use the first letter in uppercase only for **Class** and not for **methods**.
> So please change name or think up something else. For me also `_millify` It's good, but please don't use capital letter

Definitely did not forget that... Sorry!